client/src/components/About/about.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+
+class About extends Component {
+
+    render() {
+        const imageStyle = {
+            width: 400
+        }
+        return (
+            <div>
+                <div className="section center">
+                <h3>About Us</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vulputate ultrices ex eget dignissim. Donec sed molestie dolor. Nam nec nulla maximus, vestibulum mi non, sollicitudin orci. Integer auctor nulla non risus auctor aliquam sit amet quis nunc. Vivamus sit amet neque gravida, semper dui quis, tristique lacus. Pellentesque nec ipsum libero. Aenean id dapibus dui, et hendrerit purus.</p>
+                </div>
+                <br /><br />
+                    <div className="row center">
+                        <div className="col s12 m4">
```

Comment:
You have 3 div blocks here that are generally the same component. Extract this out to its own component and send down the props to that individual component of what it needs. This return of the render() can be reduced dramatically with 30-40 less lines of code.